pkg/util/bandwidth/interfaces.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bandwidth
+
+import "github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource"
+
+type BandwidthShaper interface {
+	// Initialize bandwidth shapping for an interface (e.g. 'eth0', or 'cbr0')
+	InitializeInterface(iface string) error
+	// Limit the bandwidth for a particular CIDR on a particular interface
+	//   * upload and download are in bits/second
+	//   * cidr is expected to be a valid network CIDR (e.g. 1.2.3.4/32 or 10.20.0.1/16)
```

Comment:
nit: since these are strings, put the examples in quotes to be consistent with next line.